carbon-design-system/carbon-addons-iot-react
# feat(tableviewdropdown): add prop for custom title tooltip

Closes #3534

**Summary**

- Add option to set custom browser title tooltips for dropdown items

**Change List (commits, features, bugs, etc)**

- Add prop for custom tooltips
- Add new story for tooltips
- Update docs and tests

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3643--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table-user-view-management-tableviewdropdown--with-custom-tooltips)
- Select some item from the dropdown
- Verify custom tooltips in open and closed states
- Enable `selectedViewEdited` in the knobs area
- Select some list item with custom tooltip
- Hover on the input field
- Verify that custom tooltips reflects edited state by adding `- Edited` text to the tooltip

https://user-images.githubusercontent.com/112560065/204775234-06f487ad-bf9c-45f5-9180-4826c4b3dc13.mov

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
